packages/web3-providers/src/token-price/index.ts

```diff
@@ -0,0 +1,21 @@
+import type { CurrencyType } from '@masknet/web3-shared-base'
+import { courier } from '../helpers'
+import type { TokenPriceBaseAPI } from '../types'
+
+const URL_BASE = 'https://api.coingecko.com/api/v3'
+
+export class TokenPriceAPI implements TokenPriceBaseAPI.Provider {
```

Comment:
The unit of web3-provider is one provider like debank / opensea, not someone business name.